operator/Dockerfile

```diff
@@ -1,4 +1,10 @@
-FROM docker.elastic.co/wolfi/python:3.12-dev@sha256:2f10787cc037b197f6a5d1a031c31706ae59642708c8b3b5ddfd3fa79605e9ad AS build
+ARG PYTHON_GLIBC_IMAGE="docker.elastic.co/wolfi/python"
+ARG PYTHON_GLIBC_IMAGE_VERSION="3.12-dev@sha256:2f10787cc037b197f6a5d1a031c31706ae59642708c8b3b5ddfd3fa79605e9ad"
```

Comment:
I'm not sure whether renovate will support this case

Reply:
ATM it's always trying to bump to 3.13 so I am not applying renovate patches